email-ballerina/Ballerina.toml

```diff
@@ -0,0 +1,39 @@
+[project]
+org-name = "ballerina"
+version = "0.7.0"
+
+[platform]
+target = "java8"
+
+    [[platform.libraries]]
+    artifactId = "java.mail"
+    version = "1.6.2"
+    path = "./lib/javax.mail-1.6.2.jar"
+    groupId = "javax.mail"
+
+    [[platform.libraries]]
+    artifactId = "greenmail"
+    version = "1.5.11"
+    path = "./lib/greenmail-1.5.11.jar"
+    groupId = "com.sun.mail"
+
+    [[platform.libraries]]
+    artifactId = "testng"
+    version = "6.13.1"
+    path = "./lib/testng-6.13.1.jar"
+    groupId = "org.testng"
+
+    [[platform.libraries]]
+    artifactId = "email"
+    version = "0.7.0-SNAPSHOT"
+    path = "../email-native/build/libs/email-native-0.7.0-SNAPSHOT.jar"
```

Comment:
We can parameterize this version to the project version

Reply:
Parameterized.